src/cargo/ops/cargo_install.rs

```diff
@@ -175,7 +175,11 @@ fn install_one(root: &Filesystem,
 
     let ws = match overidden_target_dir {
         Some(dir) => Workspace::ephemeral(pkg, config, Some(dir), false)?,
```

Comment:
Shouldn't the optional deps flag be set for this as well?

Reply:
ephemeral() already sets that internally